public/js/lesson-lists.js

```diff
@@ -0,0 +1,50 @@
+$(function() {
```

Comment:
Wondering if it would be worthwhile to add this same functionality to the projects page?

Reply:
Yeah, let's maybe test this out for a bit, see if we like it and then use it for projects too?